src/helper_functions.py

```diff
@@ -0,0 +1,23 @@
+"""
+This file contains some auxiliary functions
+"""
+from __future__ import annotations
+import logging
+from typing import Optional
+
+
+def get_logger(filename: Optional[str] = None) -> logging.Logger:
+    """
+    get_logger() returns a logger object that can write to a file, terminal or only file if needed.
+    """
+    logging.basicConfig(format="%(levelname)s - %(message)s")
```

Comment:
We had mentioned at some point that by default everything is redirected to `stderr`, which is not ideal. I think we want everything that is not an error to show up in `stdout`.

Reply:
I've updated the function - do we still want to write to a file?